multiqc/modules/vcftools/tstv_summary.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse TsTv by summary output from vcftools TsTv-summary """
+
+import csv
+import logging
+from collections import defaultdict, OrderedDict
+from multiqc.plots import bargraph
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class TsTvSummaryMixin():
+
+    def parse_tstv_summary(self):
+        """ Create the HTML for the TsTv summary plot. """
+
+        self.vcftools_tstv_summary = dict()
+        for f in self.find_log_files('vcftools/tstv_summary', filehandles=True):
+            d = {}
+            for line in f['f'].readlines()[1:]: # don't add the header line (first row)
+                key = line.split()[0] # taking the first column (MODEL) as key
+                val = int(line.split()[1]) # taking the second column (COUNT) as value
+                d[key] = val
+            self.vcftools_tstv_summary[f['s_name']] = d
+
+        # Filter out ignored sample names
+        self.vcftools_tstv_summary = self.ignore_samples(self.vcftools_tstv_summary)
+
+        if len(self.vcftools_tstv_summary) == 0:
+            return 0
+
+        # Specifying the categories of the bargraph
+        keys = OrderedDict()
+        keys['AC'] = { 'name': 'AC' }
+        keys['AG'] = { 'name': 'AG' }
+        keys['AT'] = { 'name': 'AT' }
+        keys['CG'] = { 'name': 'CG' }
+        keys['CT'] = { 'name': 'CT' }
+        keys['GT'] = { 'name': 'GT' }
+        keys['Ts'] = { 'name': 'Ts' }
+        keys['Tv'] = { 'name': 'Tv' }
+
+        pconfig = {
```

Comment:
I think that this plot is probably most useful to compare variations in the types of SNPs rather than the counts, right? So probably good to add `'cpswitch_c_active': False` to the config here so that the stacked percentage view is shown by default (counts can still be seen with the buttons).

Feel free to ignore if you think it's better as it is - I don't routinely work with this type of data..